### DISCUSSION TITLE
Windows 11 - WSL2

### DISCUSSION THREAD

**Discussion by ghost** [pystardust/ani-cli#302] [Q&A] [answered]
Is it possible to run anime-cli using wsl2 on windows 11? I'm trying, but the mpv doesn't start.

**Comment by Jupiteur** [comment, marked as answer]
In order to run ani-cli on a WSL (1 or 2), you need to install a serverX in order to run graphical apps from your WSL or to configure ani-cli to use vlc from your wsl (/mnt/c/the_path_to_your_vlc.exe instead of the linux media player)
